codice/ddf
# [2.19.x] DDF-5741 adding export and import command

#### What does this PR do?

https://github.com/codice/ddf/issues/5741
backport of https://github.com/codice/ddf/pull/5745

Ask 2 committers to review/merge the PR and tag them here.
@pklinef
@bdeining
@millerw8

How should this be tested?

1. install:profile standard
2. open up search ui, login as admin
3. set a new home location that's not USA on 2D map
4. verify preferences is created by querying solr preferences collection
5. http://localhost:8994/solr/preferences/select?q=:
6. create a /tmp/pref directory
7. export by store:export -t preferences /tmp/pref
8. verify a new file is created in the directory
9. verify file content look reasonable
10. delete preference by store:delete -t preferences
11. wait a min, then verify solr query above return none
12. store:list -t preferences also return zero result
13. import by store:import -t preferences /tmp/pref
14. verify solr is no populated as well as store:list
15. open 2D map and verify home location is what was set previously and not USA

#### What are the relevant tickets?
Fixes: #____

#### Screenshots


#### Checklist:
- [ x] Update / Add Unit Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
